builder/vsphere/iso/step_add_cdrom.go

```diff
@@ -62,6 +62,14 @@ func (s *StepAddCDRom) Run(_ context.Context, state multistep.StateBag) multiste
 		}
 	}
 
+	// Add our custom CD, if it exists
+	cd_path, ok := state.Get("cd_path").(string)
+	if ok {
+		if cd_path != "" {
+			s.Config.ISOPaths = append(s.Config.ISOPaths, cd_path)
+		}
+	}
```

Comment:
NIT, we can ignore wether the type assertion was succesful and here cd_path will always be a string :
```suggestion
	// Add our custom CD, if it exists
	if cd_path, _ := state.Get("cd_path").(string) ; cd_path != "" {
		s.Config.ISOPaths = append(s.Config.ISOPaths, cd_path)
	}
```

Reply:
Not true because CD path doesn't always exist, so it could be a nil interface.

Reply:
@SwampDragons do you have a playground example where that would be the case ? In my last example I tried to cover all cases but may be I missed something 👀 .

Reply:
🤦‍♀️ no I just can't read. You're right.